package.json

```diff
@@ -74,7 +78,7 @@
   },
   "devDependencies": {
     "@graphprotocol/client-cli": "3.0.0",
-    "@graphprotocol/graph-ts": "0.31.0",
+    "@graphprotocol/graph-ts": "^0.33.0",
```

Comment:
Can you revert `graph-ts` version to exactly "0.31.0"?
TheGraph and GQL libraries versioning are chaotic these days, and that's why I enforced precise versions, which are the only stable state I managed to get.